components/kyma-environment-broker/internal/process/provisioning/instance_id_override.go

```diff
@@ -0,0 +1,39 @@
+package provisioning
+
+import (
+	"time"
+
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/process"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/storage"
+	"github.com/kyma-project/control-plane/components/provisioner/pkg/gqlschema"
+	"github.com/sirupsen/logrus"
+)
+
+const BusolaMigratorComponentName = "busola-migrator"
+
+type InstanceIDOverrideStep struct {
+	operationManager *process.ProvisionOperationManager
+}
+
+func NewInstanceIDOverrideStep(os storage.Operations) *InstanceIDOverrideStep {
+	return &InstanceIDOverrideStep{
+		operationManager: process.NewProvisionOperationManager(os),
+	}
+}
+
+func (s *InstanceIDOverrideStep) Name() string {
+	return "InstanceIDOverride"
+}
+
+func (s *InstanceIDOverrideStep) Run(operation internal.ProvisioningOperation, log logrus.FieldLogger) (internal.ProvisioningOperation, time.Duration, error) {
+	instanceIDOverrides := []*gqlschema.ConfigEntryInput{
+		{
+			Key:   "deployment.env.instanceID",
```

Comment:
as I understand this value [not exist](https://github.com/kyma-project/kyma/blob/main/resources/busola-migrator/values.yaml#L13) yet?

Reply:
Will be added in a separate PR.